pkg/services/ngalert/api/api_alertmanager.go

```diff
@@ -228,6 +228,37 @@ func (srv AlertmanagerSrv) RouteGetSilences(c *contextmodel.ReqContext) response
 	return response.JSON(http.StatusOK, gettableSilences)
 }
 
+func (srv AlertmanagerSrv) RoutePostGrafanaAlertingConfigHistoryActivate(c *contextmodel.ReqContext, id string) response.Response {
+	confId, err := strconv.ParseInt(id, 10, 64)
+	if err != nil {
+		return ErrResp(http.StatusBadRequest, err, "failed to parse config id")
+	}
+
+	err = srv.mam.ActivateHistoricalConfiguration(c.Req.Context(), c.OrgID, confId)
+	if err == nil {
```

Comment:
Would prefer if we change this to `err != nil`. It's more idiomatic Go. There is nothing wrong with putting all the `errors.As` inside `if err != nil` 🙂